templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -73,7 +73,7 @@ func TestAcc<%= test_slug -%>(t *testing.T) {
 
 	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
-		<% if example_version == 'ga' -%>
+		<% unless example_version == 'beta' -%>
```

Comment:
We may need to verify that the run of MM is @ beta as well, since I think beta resources in higher-versioned runs will trigger this as well.